tests/System.Text.Primitives.Tests/Encoding/DecodeFromUTf16Tests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Primitives.Tests.Encoding
+{
+    public class DecodeFromUtf16Tests : ITextEncoderTest
+    {
+        private static TextEncoder utf16 = TextEncoder.Utf16;
+        private static Text.Encoding testEncoder = Text.Encoding.Unicode;
+
+        //[Fact]
+        public void InputBufferEmpty(TextEncoderTestHelper.SupportedEncoding from)
+        {
+            throw new NotImplementedException();
```

Comment:
I was providing the skeleton for tests for the different combinations. Will remove.